articles/quickstart.md

```diff
@@ -256,6 +262,7 @@ namespace Quantum.Bell
 }
 ```
 
+#### Run Quantum operations from C# driver
```

Comment:
We've up until this point been fairly inconsistent with respect to driver vs host program, but I think it'd be good if we could use the latter in preference to the former going forward.
```suggestion
#### Run quantum operations from a C# host program
```